### PR TITLE
feat: Added option for sticky display of variants panel with a single click

### LINKED ISSUE #2549: Feature Request: Variants panel that can be persistently displayed with a single click
related #2363 

Currently, the variants panel requires a long press to display, which can be challenging for first-time users to discover. It would be helpful to have a feature that allows the panel to be persistently displayed with a single click, enabling easier selection.

I’ve implemented a poc for this feature, which you can find here:
https://github.com/arnog/mathlive/compare/master…konbu310:mathlive:poc-variants-panel-popover

https://github.com/user-attachments/assets/3169c08b-841b-466c-9359-329f0faa1a5c

Thank you for creating and maintaining such an excellent library! It has been incredibly helpful.

### PR DESCRIPTION
**Pull request by konbu310**
fix #2549 

### Overview

This PR adds a `stickyVariantPanel` option to the VirtualKeyboardKeycap. When this option is enabled, clicking on a key with variants configured will display the Variants Panel without a long press. Additionally, the panel will not automatically hide.

The motivation for implementing this feature is described in #2549, and a similar request can be found in #2363.

### Notes

- Tests have not been added yet.
- Added an example: sticky-variant-panel/index.html.
- I am not entirely confident about the API naming.

### CODE REVIEW

File: src/public/virtual-keyboard.ts [hunk 1]
```diff
@@ -98,6 +98,9 @@ export interface VirtualKeyboardKeycap {
 
   /** Name of the layer to shift to when the key is pressed */
   layer: string;
+
+  /** Open variants panel without long press and does not close automatically */
+  popover: boolean;
```
Comment (arnog) [inline src/public/virtual-keyboard.ts:103]: I would probably go with something a bit more descriptive. Maybe `stickyVariantPanel`?

Comment (konbu310) [reply src/public/virtual-keyboard.ts:103]: That's a good name.

File: src/virtual-keyboard/utils.ts [hunk 2]
```diff
@@ -1135,19 +1135,21 @@ function handlePointerDown(ev: PointerEvent) {
   }
 
   if (keycap.variants) {
-    if (pressAndHoldTimer) clearTimeout(pressAndHoldTimer);
```
[the remark this answers is not in the document]
Comment (konbu310) [reply src/virtual-keyboard/utils.ts:1138]: It seems like I mistakenly removed it, so I’ll restore it.

**Review by arnog — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**